capstone/src/main/resources/static/event.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <title> Create an Event </title>
+    <script src="script.js"> </script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  
+  <body>
+
+    <div class="menu">
+      <nav>
+        <a href="map.html"> Campus Map </a>
+      </nav>
+    </div>    
+
+    <h1>Create an Event </h1>
+
+    <div class="event-form">
+      <form id="form" method="POST">
+
+        <label for="organizationID"> OrganizationID: </label>
+        <input type="number" id="organizationID" name="organizationID"> 
+        <br><br>
+
+        <label for="organizationName"> Organization Name: </label>
+        <input type="text" id="organizationName" name="organizationName">
+        <br><br>
+
+        <label for="university"> University: </label>
+        <input type="text" id="university" name="university">
+        <br><br>
+
+        <label for="eventTitle"> Event Title: </label>
+        <input type="text" id="eventTitle" name="eventTitle">
+        <br><br>
+
+        <label for="eventTime"> Event Time: </label>
+        <input type="time" id="eventTime" name="eventTime">
```

Comment:
It looks like this only allows users to input a time -> we should probably also have date input along with a start and end time